spec/index.bs

```diff
@@ -1469,8 +1475,28 @@ GPUDevice includes GPUObjectBase;
         The [=device=] that this {{GPUDevice}} refers to.
 </dl>
 
-{{GPUDevice}} has the methods listed in its WebIDL definition above, which are defined elsewhere in
-this document.
+{{GPUDevice}} has the methods listed in its WebIDL definition above.
+Those not defined here are defined elsewhere in this document.
+
+<dl dfn-type=method dfn-for=GPUDevice>
+    : <dfn>destroy()</dfn>
+    ::
+        Destroys the [=device=].
+
+        <div algorithm=GPUDevice.destroy()>
+            **Called on:** {{GPUDevice}} |this|.
+
+            1. Make |this|.{{GPUDevice/[[device]]}} [=invalid=].
+
+            Note: This does **not** resolve |this|.{{GPUDevice/lost}}.
```

Comment:
What's the reasoning behind this?

Reply:
I don't think it's useful to send a signal back to the app in response to it destroying the device. `.lost` is not a promise you would `await` in the middle of some logic, so there's no danger in leaving it unresolved forever - its resolution is meant to be a signal to recover from device loss. If we did resolve it on destroy, then apps would have to go out of their way to prevent their own explicit-destroy from triggering recovery logic.